lms/djangoapps/teams/models.py

```diff
@@ -134,3 +160,19 @@ def user_in_team_for_course(cls, user, course_id):
             False if not
         """
         return cls.objects.filter(user=user, team__course_id=course_id).exists()
+
+    @classmethod
+    def update_last_activity(cls, user):
+        """Set the `last_activity_at` for both this user and their team."""
+        try:
+            membership = cls.objects.get(user=user)
```

Comment:
There can be multiple memberships for the same user (both across courses and when a course allows more than one team per course). We need to send more information about what is being edited.